content/fundamentals/module-reference.md

```diff
@@ -191,17 +191,32 @@ const catsRepository = await this.moduleRef.resolve(CatsRepository, contextId);
 
 #### Instantiating classes dynamically
 
-To dynamically instantiate a class that wasn't previously registered as a provider, use the module reference's `create()` method.
+To dynamically instantiate a class that wasn't previously registered as a provider but depends on other providers use the module reference's `create()` method.
```

Comment:
Do these other providers need to be available in the current module's scope (i.e. Logger needs to be in the `providers` array)?

Reply:
@jmcdo29 Yes

Reply:
I had a feeling that was the case. It should probably be explicitly mentioned if this is going to move forward.